src/main/java/analysis/GoblintMessages.java

```diff
@@ -116,18 +116,22 @@ public List<GoblintAnalysisResult> convert() throws MalformedURLException {
 
 
     public GoblintAnalysisResult createGoblintAnalysisResult() throws MalformedURLException {
+        int column = multipiece.loc.column < 0 ? 1 : multipiece.loc.column - 1;
```

Comment:
Based on `multipiece.loc.column - 1` it seems like when Goblint returns column 1 to mean the beginning of the line, then for Magpie it gets converted to column 0. Shouldn't the negative case also return 0 then?